### PR TITLE
Introduction VXOR

### PR DESCRIPTION
**Pull request by AndreyKortosov**
I need add picture to the article

### CODE REVIEW

**Comment by WRRicht3r** [comment]
Hi, why is the file empty?
Also, the token logo is available [here](https://github.com/sora-xor/sora-branding/blob/master/tokens/SORA/svg/vXOR.svg)

**Comment by AndreyKortosov** [comment]
@WRRicht3r Hi
Good question, I've re-pushed the article

File: src/vxor.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      content: SORA VXOR token, SORA network, liquidity providing, decentralized economy, vesting, vested token, buy-back-and-burn mechanism
+---
+
+# VXOR Token Overview
+
+<center><img src=".gitbook/assets/vxor.svg" width="250"></center>
+
+VXOR, or SORA Vested XOR, is a new token designed to foster productive economic growth within the SORA ecosystem. VXOR encourages long-term commitment from users by offering benefits tied to the vesting of XOR. This unique token aligns with SORA's goal of building a stable and incentivized community of builders and participants, including national entities.
+
+# Key Features:
```
Comment (WRRicht3r) [inline src/vxor.md]:
```suggestion
## Key Features:
```

File: src/vxor.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+- Vesting Mechanism with Buy-Back-and-Burn: The VXOR mechanism uses 99% of the daily remint of XOR to buy back and burn VXOR, reducing supply and increasing value for vested participants. This mechanism essentially acts as a vesting tool to reward long-term holders.
```
Comment (WRRicht3r) [inline src/vxor.md]:
```suggestion
- A vesting Mechanism with Buy-Back-and-Burn: The VXOR mechanism uses 99% of the daily remint of XOR to buy back and burn VXOR, reducing supply and increasing value for vested participants. This mechanism essentially acts as a vesting tool to reward long-term holders.
```

File: src/vxor.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+- Encouraging Economic Expansion: Weekly calculations of the general price trend for XOR influence VXOR rewards. A positive trend results in additional VXOR being given to VXOR stakers and XOR-VXOR LP (Liquidity Provider) farmers. This mechanism is designed to align rewards with the growth in XOR’s market performance, encouraging builders and XOR-using countries to support SORA’s token economy actively.
```
Comment (WRRicht3r) [inline src/vxor.md]:
```suggestion
- Economic Expansion Encouragement: Weekly calculations of the general price trend for XOR influence VXOR rewards. A positive trend results in additional VXOR being given to VXOR stakers and XOR-VXOR LP farmers. This mechanism is designed to align rewards with the growth in XOR’s market performance, encouraging builders and XOR-using countries to actively support SORA’s token economy.
```

File: src/vxor.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+- Support for SORAized Countries: Beyond individual builders, SORAized countries—nations that integrate XOR into their economies—can also receive VXOR through on-chain governance to bolster their economic expansion. The aim is to apply economic principles (ΔMV = ΔPY) to support increased purchasing power and economic growth in XOR-using regions.
+- Sustainable XOR Supply Reduction: VXOR’s integration into SORA’s “v3 tokenomics” continues the gradual reduction in XOR supply, promoting a stable and long-term growth strategy for the SORA ecosystem.
+
+![](.gitbook/assets/sora_vxor_infinite_loop.png)
+
+# In Summary
```
Comment (WRRicht3r) [inline src/vxor.md]:
```suggestion
## In Summary
```

File: src/vxor.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+VXOR is a vested token with a buy-back-and-burn mechanism designed to support SORA’s economic vision. It offers an innovative way for both individual participants and XOR-integrated nations to contribute to, and benefit from, the growth of the SORA ecosystem.
+
+# More Details
```
Comment (WRRicht3r) [inline src/vxor.md]:
```suggestion
## More Details
```

**Review by WRRicht3r — CHANGES_REQUESTED** [review]

File: src/vxor.md (hunk 1 above)
Comment (WRRicht3r) [reply src/vxor.md]: Only one H1 per page please

**Review by WRRicht3r — APPROVED** [review]

**Pull request merged**